### DISCUSSION TITLE
Are the ICD codes annotated by doctors or secretaries?

### DISCUSSION THREAD

**Discussion by JoakimEdin** [MIT-LCP/mimic-code#1527] [MIMIC-General]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
Some cases are annotated with ICD codes that aren't described in the discharge summary. My hypothesis is that a doctor has annotated the codes and forgotten to document them in the discharge summary. If a secretary is annotated the codes this hypothesis can't be true. Are the ICD codes annotated by doctors or secretaries?

**Comment by alistairewj** [comment]
I would say neither! There are trained coders who review all patient notes and assign ICD codes. If necessary they will contact the associated providers to confirm a diagnosis, but they will require the provider to input that information into a signed note as they are only allowed to code diagnoses from signed notes.

The discharge summary is one of the notes they review (and one of the most comprehensive), but not the only one. It's possible the ICD code was assigned based on another note not publicly available in MIMIC currently.

**Comment by abhinayvyas** [comment]
Thanks, this helps. As an extension to above question, do we have any additional details to know if codes submitted by coders/providers been accepted by payer? I have been looking at a few coma codes (R40.2*) and have noticed lack of substantial information to support documentation for billable ICD codes. Reference Hadm_ids: 22747866,  26829618.
